### ISSUE TITLE
Remove a parameter

### ISSUE THREAD

**Issue by thujohn** [jasonlewis/enhanced-router#5]
Hi.

You said

> It's important to note that the locale is actually given to each route as a parameter. The parameter is also given to every method of every controller that is within the group. When your route requires a parameter of its own it will be given after the prefix parameter.

If I have a big site and I don't want to modify all my methods to add the $locale parameter to each methods how can I remove it ?

**Comment by jasonlewis** [comment]
It's actually out of my control as it's done within Symfony from memory. Sorry. Off the top of my head I don't think there is a way to get around it without hacking at Symfony.

**Comment by thujohn** [comment]
So the best solution would be the return of the Call Controller from L3...

**Comment by jasonlewis** [comment]
Mm, that would be helpful. Don't know if that would happen though. Maybe. :smile:

**Comment by thujohn** [comment]
Maybe you know a solution to call a method from the routes ?

**Comment by Tickthokk** [comment]
I installed this and it was great, but in the end I couldn't use it for this reason.  In every controller I had to have parameters for the first two segments, as I was using them as a variable, like this:

```
Route::group(array('prefix' => '{location}/{company}), function()
{
```

In App::before filter, I'm doing stuff with segment's 1 and 2 anyway, but after that I don't need them.  They are Locale and the Slug of a company.  So in this case those segments are variables, but (in essence) I don't care what they are after the filter (I'm setting them as configuration options to use later, and they're checked against a set of allowed languages/etc).

This was my solution:

```
$language = Request::segment(1);
$company = Request::segment(2);

Route::group(array('prefix' => $language . '/' . $company), function()
{

```

It's still dynamic, but the route thinks it was static.

I hope this helps someone.  Keep up the good work on this repo, I definitely think it has its uses :+1:
